packages/syft/src/syft/store/document_store.py

```diff
@@ -699,7 +699,7 @@ def update(
         credentials: SyftVerifyKey,
         obj: BaseStash.object_type,
         has_permission=False,
-    ) -> Optional[Result[BaseStash.object_type, str]]:
+    ) -> Result[BaseStash.object_type, str]:
```

Comment:
@shubham3121 Fixed the type here. This just returns a `Result` not an `Optional[Result]`.